wye/templates/email_messages/organisation/additional_member.txt

```diff
@@ -0,0 +1,10 @@
+Hey {{full_name}},
+
+You've added {{ new_member_name }} to your organization. {{ new_member_name }} can now access your organization dashboard and manage workshops. Visit your profile (link-to-profile) to view or update your organization details.
+
```

Comment:
@vnbang2003 Sorry, that profile refers to the organization's profile page. When a new member is added to an organization, he/she can edit the organization details right?